site/source/tutorials/9001_dojo1_dijits/index.md

```diff
@@ -0,0 +1,187 @@
+
+{% section %}
+
+## Placing Dijits
+
+{% task 'Adding to a widget\'s render function.' %}
+
+Now that we have imported our wrapped Dijits into the module we want to use them in, we need to add them to the render function.
+
+{% instruction 'Add imported Dijits to the `WorkerForm` widget\'s render function:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:44-74 %}
+
+{% aside 'Information' %}
+The <em>properties</em> for a wrapped Dijit are equivalent to the <em>params</em> that can be passed when constructing a Dijit.  For more information on what parameters are possible on Dijits, you can refer to the [Dojo 1 API documentation](https://dojotoolkit.org/api/).
+{% endaside %}
+
+Once Dijits are wrapped, they behave very similar to Dojo 2 widgets.  Coupled with the `dojo-typings` TypeScript typings, you get code completion as well when specifying the _properties_ and TypeScript should help ensure the values of the properties are valid.
+
+The one restriction to be aware of is that any wrapped Dijit can only have other wrapped Dijit's as children.  This is because all Dojo 2 widget's DOM is represented virtually, but all Dojo 1 Dijits directly manage their own DOM.  Having virtual DOM children is not something a Dijit is capable of dealing with.
+
+{% task 'Setting a Dojo 1 theme.' %}
+
+When we configured the _externals_ for our project, we included the main CSS for our theme.  Most Dijits will not render properly without a theme being applied to them.  There are many different ways to apply a Dojo 1 theme, but the most straight forward is to set the theme class name on the `<body>` of the application document.
+
+{% instruction 'Add theme class name to `src/index.html`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/index.html' lines:6-8 lang:html %}
+
+You should now have an application that will provide you with a form that takes a first name, last name, and e-mail address in order to add another _card_ to the application.
+
+{% section %}
+
+## Summary
+
+In this tutorial, we learned how to setup the build system to include Dojo 1 as an external dependency.  We learned how to import Dojo 1 based Dijits into a Dojo 2 project and wrap them so they can be integrated into the Dojo 2 widgeting system.  We also learned how to apply a Dojo 1 theme so our Dijits have a proper look and feel.
+
+If you would like, you can download the completed [demo application](../assets/9001_dojo1_dijits-finished.zip) from this tutorial.
+
+This tutorial is far from exhaustive on the subject, but is intended to provide the basics of using Dojo 1 Dijits in a Dojo 2 application.  In most cases we still feel it is better to use or create Dojo 2 widgets than to leverage Dojo 1 Dijits.  We think the main use case is when you want to incrementally migrate an existing application to Dojo 2, being able to _wrap_ custom Dijits can be really useful.
```

Comment:
In most cases we still feel it is better to use or create Dojo 2 widgets than to leverage Dojo 1 Dijits. -> It is ideally better to use or create Dojo 2 widgets than to leverage Dojo 1 Dijits.